app/controllers/todo_items_controller.rb

```diff
@@ -1,15 +1,15 @@
 class TodoItemsController < ApplicationController
   def index
     begin
-      @todo_list = TodoList.find_by(id: params[:todo_list_id])
+      @todo_list = TodoList.find(params[:todo_list_id])
     rescue ActiveRecord::RecordNotFound
-      redirect_to new_todo_list_path
+      redirect_to new_todo_list_path and return
```

Comment:
'Return' halts the method and nothing else will occur after that. Technically it's not needed here, as there is no code following this line. However, it is necessary on line 72 of the controller, because there is a redirect called after that line that I don't want to run.